### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by tejks**

### CODE REVIEW

File: app/src/components/Navigation/LeftNavbar.svelte [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	import WalletMultiButton from '../Wallet/WalletMultiButton.svelte';
+	import BoxIcon from './NavbarIcons/BoxIcon.svelte';
+	import HomeIcon from './NavbarIcons/HomeIcon.svelte';
+	import SendIcon from './NavbarIcons/SendIcon.svelte';
+	import TrackIcon from './NavbarIcons/TrackIcon.svelte';
+
+	$: currentPage = $page.url.pathname;
+	$: isNavbarOpen = false;
+
+	const navigation = [
```
Comment (tommysr) [inline app/src/components/Navigation/LeftNavbar.svelte:15]: I would fit somewhere "+" for adding shipments, i think after the home icon. Then package icon will be listing shipments for forwarders, but anyone can access listing too.  These names are kinda tricky, doesn't leak any functionality which is behind them.

File: app/src/routes/shipmentsMap/+layout.svelte [hunk 2]
```diff
@@ -122,33 +120,20 @@
...
+<div
+	class="hidden md:block absolute z-10 w-3/4 md:w-1/3 xl:w-1/4 left-1/2 transform -translate-x-1/2 top-4"
+>
+	<div class="m-3 p-0.5 rounded-full bg-gradient-to-r from-primary to-secondary">
+		<label for="name" class="sr-only">Name</label>
+		<input
+			class="px-3 py-1.5 w-full rounded-full bg-background focus:outline-none text-sm md:text-lg"
+			type="text"
+			id="name"
+			placeholder="Search"
```
Comment (tommysr) [inline app/src/routes/shipmentsMap/+layout.svelte:132]: i can't find it on mobile, is it there somewhere?

Comment (tommysr) [reply app/src/routes/shipmentsMap/+layout.svelte:132]: i mean no searchbar on mobile

**Review by tommysr — COMMENTED** [review]
Check this searchbar problem, and add icon for creation of shipment.

File: app/src/components/Navigation/LeftNavbar.svelte (hunk 1 above)
Comment (tommysr) [reply app/src/components/Navigation/LeftNavbar.svelte:15]: e.g shippers for package have no additional meaning, maybe shipments is more appropriate

**Comment by tommysr** [comment]
But overall looks really good.

File: app/src/routes/shipmentsMap/+layout.svelte (hunk 2 above)
Comment (tejks) [reply app/src/routes/shipmentsMap/+layout.svelte:132]: I know that. I'm still thinking how to do it.

Comment (tejks) [reply app/src/routes/shipmentsMap/+layout.svelte:132]: Becouse it should be in Navbar, but this complicates the search you've already created.

File: app/src/components/Navigation/LeftNavbar.svelte (hunk 1 above)
Comment (tejks) [reply app/src/components/Navigation/LeftNavbar.svelte:15]: I've been thinking about it and I have a few ideas.

File: app/src/routes/shipmentsMap/+layout.svelte (hunk 2 above)
Comment (tommysr) [reply app/src/routes/shipmentsMap/+layout.svelte:132]: How so? Keyboard events?

Comment (tommysr) [reply app/src/routes/shipmentsMap/+layout.svelte:132]: I would leave it for later, just open issue to not forget it.

**Comment by matzayonc** [comment]
Looks sick overall, great work guys!

One thing I noticed is that reload on a subpage doesn't load. Is that expected?

**Review by matzayonc — APPROVED** [review]

**Comment by tejks** [comment]
Recent navigation changes. If looks good, I'll merge it.

**Comment by tommysr** [comment]
> Looks sick overall, great work guys!
> 
> One thing I noticed is that reload on a subpage doesn't load. Is that expected?

I guess it is buffer dependency related

**Review by tommysr — APPROVED** [review]

**Pull request merged**